.travis.yml

```diff
@@ -109,12 +109,38 @@ jobs:
           branch: master
           repo: denoland/deno
         skip-cleanup: true
+    
+    - name: "cargo release linux arm64"
+      os: linux
+      dist: xenial
+      script:
+      - rustup target add aarch64-unknown-linux-gnu
+      - sudo apt update
+      - |-
+        sudo apt -yq --no-install-suggests --no-install-recommends install
+        g++-5-aarch64-linux-gnu gcc-5-aarch64-linux-gnu g++-5-multilib
+        libc6-arm64-cross
+      - build/linux/sysroot_scripts/install-sysroot.py --arch=arm64
+      - export DENO_BUILD_ARGS="target_cpu=\"arm64\" v8_target_cpu=\"arm64\""
+      - export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - export CC_aarch64_unknown_linux_gnu="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --locked
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --tests --locked
+      - |-
+        mv $(find $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release
+        -maxdepth 1 -type f -executable -regextype sed -regex
+        '.*/[a-z_]*-[0-9a-f]*') 
+        $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release/test_rs
```

Comment:
This requires tools/http_server.py is running concurrently. I think it's not?

Remove this

Reply:
This one should make more sense now that the backslashes are in.

Reply:
I'd rather remove it for now. We will revisit. The arm build is going to need a whole other pass for testing before we can release it - so I'd rather just keep it simple now.